### DISCUSSION TITLE
Hxjzaksksjakkk

### DISCUSSION THREAD

**Discussion by hhhh853** [TeamNewPipe/NewPipe#11535] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

 ndjjzkzmsdhfjsjsh

### Additional information

Ncjdekxjdjcjccjj

**Comment by opusforlife2** [comment]
Is this how you try to open a demon portal?
